src/pages/Donate/DonateForm.vue

```diff
@@ -25,9 +25,7 @@
 					v-model="isMonthly"
 					v-kv-track-event="['Donate form', 'toggle-monthly-donation', 'Make a monthly donation.']"
 				>
-					<template #after>
-						Make a monthly donation.
-					</template>
+					Make a monthly donation.
```

Comment:
Guessing this just works and looks right without using the `#after` slot...

Reply:
Yeah, part of the earlier change to kv-base-input